appeals/e2e/cypress/support/appealsApiClient.js

```diff
@@ -0,0 +1,41 @@
+// @ts-nocheck
+import axios from 'axios';
```

Comment:
If I understand correctly, cypress code runs in a browser context? If so, we could use native `fetch` instead.

Reply:
Great point, one less dependency with minimal changes